pep-0676.rst

```diff
@@ -33,15 +35,15 @@ This PEP provides a specification for self-contained rendering of PEPs. This
 would:
 
 * reduce the amount of distributed configuration for supporting PEPs
-* enable quality-of-life improvements for those who read, write, and review
-  PEPs
+* enable quality-of-life improvements for those who read, write and review PEPs
 * solve a number of outstanding issues, and lay the path for improvements
 * save volunteer maintainers' time
 
-We propose that PEPs are accessed through peps.python.org_ at the top-level
-namespace (for example `peps.python.org/pep-0008/`_), and that all custom
+We propose that PEPs are accessed through ``peps.python.org`` at the top-level
+namespace (for example, ``peps.python.org/pep-0008/``), and that all custom
```

Comment:
Let's keep these as real links because (fingers crossed!) one day this PEP will implemented and future PEP readers can click the real links to see the output.


```suggestion
We propose that PEPs are accessed through `peps.python.org/pep-0008/`_ at the top-level
namespace (for example, `peps.python.org/pep-0008/`_), and that all custom
```

Reply:
I didn't like the specter of deliberately broken links and figured they would make sense as verbatim literals instead, but fair enough. I'll need to revert the removals in the references list too.